### PR TITLE
Make the show_urls command honor the --no-color argument.

### PR DESCRIPTION
**Pull request by tysonclugg**
Pretty straight forward, includes test cases.

### CODE REVIEW

**Comment by tysonclugg** [comment]
@trbs Let me know if you want any more work done on this branch, or if you'd like me to rebase the commits according to your preferences.

File: django_extensions/management/commands/show_urls.py [hunk 1]
```diff
@@ -49,7 +49,10 @@ def handle(self, *args, **options):
         if args:
             appname, = args
 
-        style = color_style()
+        if options['no_color']:
```
Comment (trbs) [inline django_extensions/management/commands/show_urls.py]: Can you do ```options.get('no_color')``` please ?

**Comment by tysonclugg** [comment]
@trbs: Done, tests passing.  :-)

**Comment by trbs** [comment]
and merged :)

Thanks !!

**Pull request merged**